src/dataflow-types/src/types/sources.rs

```diff
@@ -24,38 +24,36 @@ use proptest::prop_oneof;
 use proptest_derive::Arbitrary;
 use prost::Message;
 use serde::{Deserialize, Serialize};
-use timely::progress::Antichain;
 use uuid::Uuid;
 
 use mz_kafka_util::KafkaAddrs;
+
 use mz_persist_types::Codec;
 use mz_repr::chrono::any_naive_datetime;
 use mz_repr::proto::{any_duration, any_uuid, TryFromProtoError};
 use mz_repr::proto::{IntoRustIfSome, ProtoType, RustType};
 use mz_repr::{ColumnType, GlobalId, RelationDesc, RelationType, Row, ScalarType};
 
-pub mod encoding;
-
 use crate::aws::AwsConfig;
 use crate::postgres_source::PostgresSourceDetails;
 use crate::DataflowError;
 
+pub mod encoding;
+
 include!(concat!(
     env!("OUT_DIR"),
     "/mz_dataflow_types.types.sources.rs"
 ));
 
 /// A description of a source ingestion
 #[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
-pub struct IngestionDescription<S = (), T = mz_repr::Timestamp> {
-    /// Source collections made available to this ingestion.
-    pub source_imports: BTreeMap<GlobalId, S>,
+pub struct IngestionDescription<S = ()> {
     /// The source identifier
     pub id: GlobalId,
     /// The source description
     pub desc: SourceDesc,
-    /// The initial `since` frontier
-    pub since: Antichain<T>,
```

Comment:
why are we move this out of the description into the `CreateSources` command? am I missing something, or are they still always available at the same time?

Reply:
Comment is outdated now, right?